src/verifiers/revocationStore/openAttestationRevocationStore.ts

```diff
@@ -0,0 +1,34 @@
+import { getData, utils, v2, WrappedDocument } from "@govtechsg/open-attestation";
+import { VerificationFragmentType, Verifier } from "../../types/core";
+import { OpenAttestationRevocationCode } from "../../types/error";
+import { openAttestationEthereumDocumentStoreRevoked } from "../documentStoreRevoked/openAttestationEthereumDocumentStoreRevoked";
+
+const name = "OpenAttestationRevocationStore";
+const type: VerificationFragmentType = "DOCUMENT_STATUS";
+export const openAttestationRevocationStore: Verifier<WrappedDocument<v2.OpenAttestationDocument>> = {
+  skip: () => {
+    return Promise.resolve({
```

Comment:
wont this resolve be redundant? or can just add `async` to the function

Reply:
could, just coding style :p (both are equivalent)